utils/luafunc_dup.sh

```diff
@@ -0,0 +1,16 @@
+#!/usr/bin/bash
+
+LST=`tempfile`
+res=0
+trap 'rm -f $LST ; exit $res' EXIT
+
+for arg in $@ ; do
+   grep '^ \* @luafunc' $arg | sort | uniq -c | grep -v "^ *1 " > $LST
+
+   if [ -s "$LST" ] ; then
+      echo "$arg:"
+      cat $LST
```

Comment:
```suggestion
      cat "$LST"
```
More potential resplitting.

Reply:
LST is a tempfile name, so shouldn't contain spaces.
But better safe than sorry, let us do this.